guis/desktop/gui.py

```diff
@@ -192,11 +193,56 @@ def create_map(self):
         url = wx.FileSystem.FileNameToURL(MAP_HTML)
         self.browser.LoadURL(url)
         self.right.Add(self.browser, 1,
-                       wx.EXPAND | wx.RIGHT | wx.LEFT | wx.BOTTOM, 11)
+                wx.EXPAND | wx.RIGHT | wx.LEFT, 10)
+
+
+    def create_logical_buttons(self):
+        # Default
+        self.logical_operator = "and"
+
+        width = 400 // 11
+
+        # Horizontal sizer for spacing
+        btm_right = wx.BoxSizer(wx.HORIZONTAL)
+
+        # AND
+        self.and_btn = wx.RadioButton(self.panel, label="and",
```

Comment:
`_button`? It's more personal preference, so then need to revert my change `_btn` => `_button`.